app/src/org/commcare/dalvik/activities/EntitySelectActivity.java

```diff
@@ -788,6 +791,19 @@ private void select() {
         setResult(RESULT_OK, i);
         finish();
     }
+
+    // CommCare-159503: implementing DetailCalloutListener so it will not crash the app when requesting call/sms

```

Comment:
This code is open source but FogBugz tickets are internal. Better to explain the bug than refer to a ticket.